source/views/guides/getting-started.html.slim

```diff
@@ -37,9 +37,15 @@ main
                   cd diesel_demo
 
         markdown:
-          First, let's add Diesel to our dependencies. We're also going to use a
-          tool called [`.env`][dotenv-rust] to manage our environment variables
-          for us. We'll add it to our dependencies as well.
+          First, let's add Diesel to our dependencies. The features option lists the
+          databases we want to use. The current options are: `postgres`, `mysql` and
+          `sqlite`. You can list multiple.
+
+          Please note that you may need to install the corresponding client libraries
+          for your operating system.
```

Comment:
Can you mention common error messages so people can search for them? Something like

> If a client library is missing, you get an error message like `library not found for -lmysqlclient` (for MySQL).